frontend/src/component/AddIcon/AddIcon.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  SafeAreaView,
+  StyleSheet,
+  Dimensions,
+  TouchableOpacity,
+  Image,
+} from 'react-native';
+import React from 'react';
+
+const HEIGHT = Dimensions.get('window').height;
+
+const AddIcon = () => {
+  return (
+    <SafeAreaView>
+      <TouchableOpacity >
+        <Image
```

Comment:
* Create a circular view (With PRIMARY_COLOR). and then add a white pen icon from **react-native-vector-icons** at the center of the circle.
* Try not to use any static image like PNG, jpg, or jpeg, because sometimes, it will not display on the phone (for some devices).  you can either use SVG images or icons from any react-native libraries.